.circleci/config.yml

```diff
@@ -212,6 +209,11 @@ jobs:
           command: |
             pnpm docs:link-check
             git add -A && git diff --exit-code --staged
+      - run:
+          name: '`pnpm build:charts:vendor` changes committed?'
+          command: |
+            pnpm build:charts:vendor
```

Comment:
Nitpick: I see that it takes ~9 seconds on CI, but still, have you considered running it less frequently?
Maybe use the same logic as for the `dedupe`?
https://github.com/mui/mui-x/blob/ad605ffc99fda53c8e2f6f429824dec5e428767f/.circleci/config.yml#L126-L134